atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/DbkvsPostgresTestSuite.java

```diff
@@ -41,7 +42,8 @@
         DbkvsPostgresKeyValueServiceTest.class,
         DbkvsPostgresSerializableTransactionTest.class,
         DbkvsPostgresSweeperTest.class,
-        PostgresDbTimestampBoundStoreTest.class
+        PostgresDbTimestampBoundStoreTest.class,
+        PostgresVersionCheckTest.class
```

Comment:
I looked into it - you just needed to move the test to the test folder of `atlasdb-dbkvs`.